packages/patterns/src/types.js

```diff
@@ -449,7 +449,7 @@ export {};
  *
  * @property {(subPatt: Pattern) => Pattern} eref
  * Matches any Passable that is either matched by `subPatt` or is a promise object.
- * Note that validation is immediate, so (unlike the TypeScript ERef<T>
+ * Note that validation is immediate, so (unlike the TypeScript `ERef<T>`
```

Comment:
<T> if left unquoted, will be confused with a HTML tag by vitepress, and will throw an error that looks like `SyntaxError: Element is missing end tag.`

We can see a similar error being thrown for something else in agoric-sdk repo before vitepress can process this file here: https://github.com/Agoric/documentation/actions/runs/8442634353/job/23124473938?pr=1038